throughput/src/bin/z_put_thr.rs

```diff
@@ -84,6 +95,18 @@ async fn main() {
         .into();
 
     let session = zenoh::open(config).await.unwrap();
+    let writer = if use_expr {
+        if declare_publication {
+            session.declare_publication(KEY_EXPR);
+        }
+        session.put(KEY_EXPR, value.clone())
+    } else {
+        let expr_id = session.declare_expr(KEY_EXPR).await.unwrap();
+        if declare_publication {
+            session.declare_publication(expr_id);
+        }
+        session.put(expr_id, value.clone())
+    };
```

Comment:
I'm afraid that you use the opposite in this if/else branch. 😅

Reply:
You are right. Will fix this.

Reply:
Regarding to the name pub or put, according to the architecture concept and the API design of zenoh, the data-centric view favors put than pub. As a result, I would suggest we keep it as z_put. Or, maybe we can postpone this for later discussion?

Reply:
okay